sdk/purview/azure-purview-account/README.md

```diff
@@ -0,0 +1,164 @@
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET
+
+Use the returned token credential to authenticate the client:
+
+```python
+from azure.purview.account import PurviewAccountClient
+from azure.identity import DefaultAzureCredential
+
+credential = DefaultAzureCredential()
+client = PurviewAccountClient(endpoint="https://<my-account-name>.purview.azure.com", credential=credential)
+```
+
+## Key concepts
+
+### Client
+
+This package offers request builders so you can build http requests and send these requests to the service using the `send_request` method.
+For more information on how to use request builders and our clients, see [here][request_builders_and_client].
+
+## Examples
+
+The following section shows you how to initialize and authenticate your client, then list all of your keys.
+
+- [List All Keys](#list-all-keys "List All Keys")
+
+### List All Keys
+
+```python
+from azure.purview.account import PurviewAccountClient
+from azure.identity import DefaultAzureCredential
+from azure.purview.account.rest import accounts
+from azure.core.exceptions import HttpResponseError
+
+credential = DefaultAzureCredential()
+client = PurviewAccountClient(endpoint="https://<my-account-name>.purview.azure.com", credential=credential)
+
+request = accounts.build_list_keys_request()
+
+response = client.send_request(request)
+try:
+    response.raise_for_status()
+    assert response.status_code == 200
+
+except HttpResponseError as e:
+    print(e)
+```
+
+## Troubleshooting
+
+### General
+
+The Purview Account client will raise exceptions defined in [Azure Core][azure_core] if you call `.raise_for_status()` on your responses.
+
+### Logging
+
+This library uses the standard
+[logging][python_logging] library for logging.
+Basic information about HTTP sessions (URLs, headers, etc.) is logged at INFO
+level.
+
+Detailed DEBUG level logging, including request/response bodies and unredacted
+headers, can be enabled on a client with the `logging_enable` keyword argument:
+
+```python
+import sys
+import logging
+from azure.identity import DefaultAzureCredential
+from azure.purview.account import PurviewAccountClient
+
+# Create a logger for the 'azure' SDK
+logger = logging.getLogger('azure')
+logger.setLevel(logging.DEBUG)
+
+# Configure a console output
+handler = logging.StreamHandler(stream=sys.stdout)
+logger.addHandler(handler)
+
+endpoint = "https://<my-account-name>.purview.azure.com"
+credential = DefaultAzureCredential()
+
+# This client will log detailed information about its HTTP sessions, at DEBUG level
+client = PurviewAccountClient(endpoint=endpoint, credential=credential, logging_enable=True)
+```
+
+Similarly, `logging_enable` can enable detailed logging for a single `send_request` call,
+even when it isn't enabled for the client:
+
+```python
+result = client.send_request(request, logging_enable=True)
```

Comment:
I think it would be better to switch this call to an operation call, for example

```python
result = client.list_keys(..., logging_enable=True)
```